### ISSUE TITLE
Store the chunking parameters in the model config

### ISSUE THREAD

**Issue by iiSeymour** [nanoporetech/bonito#19]
The learning rate, batch size, seed, ect are stored in the `model.toml` for each model trained, it would be useful *(#18)* to also store all the parameters given to `convert-data` in the `model.toml`. Not all of chunking parameters are available *(or can be computed)* at training time so just make the `convert-data` script dump a `conversion.toml` and read it at training time to store in the `model.toml`.
